packages/siop-oid4vp/lib/authorization-response/OpenID4VP.ts

```diff
@@ -284,7 +286,11 @@ export const assertValidVerifiablePresentations = async (args: {
     hasher?: Hasher
   }
 }) => {
-  const presentationsArray = args.presentations ? (Array.isArray(args.presentations) ? args.presentations : [args.presentations]) : []
+  const {presentations} = args
+  if (!presentations || (Array.isArray(presentations) && presentations.length === 0)) {
+    throw Error('missing presentation(s)')
```

Comment:
Promise.reject(Error('

Reply:
Also fixed, I had to add a return type : Promise<void> to allow return Promise.reject()